Apps/Src/CLI.c

```diff
@@ -1,809 +1,809 @@
- */
-void CLI_LED(int* hashTokens) {
-	switch(hashTokens[1]) {
-		case CLI_TEST_HASH:
-			for(int i = 0; i < 10; i++) {
-				BSP_Light_Toggle(RUN);
-				DelayMs(100);
-				BSP_Light_Toggle(UVOLT);
-				DelayMs(100);
-				BSP_Light_Toggle(OVOLT);
-				DelayMs(100);
-				BSP_Light_Toggle(OTEMP);
-				DelayMs(100);
-				BSP_Light_Toggle(OCURR);
-				DelayMs(100);
-				BSP_Light_Toggle(WIRE);
-				DelayMs(100);
-				BSP_Light_Toggle(WDOG);
-				DelayMs(100);
-				BSP_Light_Toggle(CAN);
-				DelayMs(100);
-				BSP_Light_Toggle(EXTRA);
-				DelayMs(100);
-			}
-			break;
-		case CLI_RUN_HASH:
-			setLED(RUN, hashTokens[2]);
-			break;
-		case CLI_OCURR_HASH:
-			setLED(OCURR, hashTokens[2]);
-			break;
-		case CLI_OTEMP_HASH:
-			setLED(OTEMP, hashTokens[2]);
-			break;
-		case CLI_UVOLT_HASH:
-			setLED(UVOLT, hashTokens[2]);
-			break;
-		case CLI_OVOLT_HASH:
-			setLED(OVOLT, hashTokens[2]);
-			break;
-		case CLI_WDOG_HASH:
-			setLED(WDOG, hashTokens[2]);
-			break;
-		case CLI_CAN_HASH:
-			setLED(CAN, hashTokens[2]);
-			break;
-		case CLI_EXTRA_HASH:
-			setLED(EXTRA, hashTokens[2]);
-			break;
-		default:
-			printf("Invalid LED command\n\r");
-			break;
-	}
-}
-
-/** CLI_CAN
- * Interacts with CAN by
- * reading and writing to bus
- * @param hashTokens is the array of hashed tokens
- */
-void CLI_CAN(int* hashTokens) {
-    uint32_t id;
-	uint8_t rxData[8];
-	uint8_t txData[8];
-	uint8_t dataLength = (strlen(hexString)/2) + (strlen(hexString)%2);	// Ceiling of length
-	for(uint8_t i = 0; i < 7; i+= 2) {
-		char tempData[2] = {hexString[i], hexString[i+1]};
-		txData[i/2] = strtol(tempData, NULL, 16);
-	}
-	switch(hashTokens[1]) {
-		case CLI_READ_HASH: 
-			while(BSP_CAN_Read(&id, rxData)){
-                for(int32_t i = 0; i < 8; i++) {
-    				printf("0x%x  ", rxData[i]);
-                }
-                printf("\r\n");
-			}
-		break;
-		case CLI_WRITE_HASH: 
-			switch(hashTokens[2]) {
-				case CLI_TRIP_HASH:
-					BSP_CAN_Write(TRIP, txData, dataLength);
-					break;
-				case CLI_CLEAR_HASH:
-					BSP_CAN_Write(ALL_CLEAR, txData, dataLength);
-					break;
-				case CLI_OFF_HASH:
-					BSP_CAN_Write(CONTACTOR_STATE, txData, dataLength);
-					break;
-				case CLI_CURRENT_HASH:
-					BSP_CAN_Write(CURRENT_DATA, txData, dataLength);
-					break;
-				case CLI_VOLTAGE_HASH:
-					BSP_CAN_Write(VOLT_DATA, txData, dataLength);
-					break;
-				case CLI_TEMPERATURE_HASH:
-					BSP_CAN_Write(TEMP_DATA, txData, dataLength);
-					break;
-				case CLI_CHARGE_HASH:
-					BSP_CAN_Write(SOC_DATA, txData, dataLength);
-					break;
-				case CLI_WATCHDOG_HASH:
-					BSP_CAN_Write(WDOG_TRIGGERED, txData, dataLength);
-					break;
-				case CLI_ERROR_HASH:
-					BSP_CAN_Write(CAN_ERROR, txData, dataLength);
-					break;
-				default:
-					printf("Invalid ID\n\r");
-					break;
-			}
-			break;
-		default:
-			printf("Invalid CAN command\n\r");
-			break;
-	}
-}
+// /* Copyright (c) 2020 UT Longhorn Racing Solar */
```

Comment:
Is there a  reason all of this is commented out?

Reply:
It doesn't compile with the new EEPROM library. I can fix the compile errors if you want